CMakeLists.txt

```diff
@@ -89,8 +118,25 @@ if (Boost_FOUND)
     endif()
 endif()
 
+# Try to find the RdKafka configuration file if present.
+# This will search default system locations as well as RdKafka_ROOT and RdKafka_Dir paths if specified.
+find_package(RdKafka ${FIND_PACKAGE_QUIET} CONFIG)
+set(RDKAFKA_TARGET_IMPORTS ${RdKafka_FOUND})
```

Comment:
The messages I'm getting while running this are very misleading:

```
-- RdKafkaConfig.cmake not found. Please set RDKAFKA_ROOT or RDKAFKA_DIR if a config file is installed. Attempting to find module instead...
-- Could NOT find RDKAFKA (missing: RdKafka_LIBRARY_DIR) 
-- Found valid rdkafka version
-- RdKafka module found.
```

It made me think it didn't find it but then it turns out it didn't find it. I don't think any warning messages should be shown unless the library is not found at all. e.g. no "Please set ...." as it makes the user think something went wrong. Also definitely not the "Could NOT find RDKAFKA" message.

Reply:
Ok I will remove the intermediate message. Alternatively I can simply print "RdKafka config file not found. Searching for module instead...". ? Would that be clearer?

Reply:
I think the best way would be to, once it found it, to print how it did it. If it didn't find it at all, then it can print the "please set ..." message so users can fix it. I don't think it matters from a user point of view whether rdkafka was found via config or via module, but I may be wrong.